jwala-services/src/main/java/com/cerner/jwala/service/media/impl/MediaServiceImpl.java

```diff
@@ -101,6 +109,46 @@ public JpaMedia create(final Map<String, String> mediaDataMap, final Map<String,
                 format("{0} does not have any root directories! It may not be a valid media file.", filename));
     }
 
+    private String checkForExistingBinary(final String filename, final String uploadedFilePath) {
```

Comment:
This method relies on the RepositoryService. Are we ok to add that dependency to JwalaUtils?